apps/explorer/src/pages/transaction-result/programmable-transaction-view/Commands.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { type ProgrammableTransactionCommand } from '@mysten/sui.js';
+
+import { Command } from '~/pages/transaction-result/programmable-transaction-view/Command';
+import { TableHeader } from '~/ui/TableHeader';
+
+interface Props {
+    commands: ProgrammableTransactionCommand[];
+}
+
+export function Commands({ commands }: Props) {
+    if (!commands?.length) {
+        return null;
+    }
+
+    return (
+        <div data-testid="programmable-transactions-commands" className="mt-8">
```

Comment:
Don't bake margin into these components, add it to the place this is used instead.